src/artefacts_demo_control/test/artefacts_demo_control/collision_test.py

```diff
@@ -85,44 +85,55 @@ def test_collsion(self):
             x_variable = float(match.group(1))
             y_variable = float(match.group(2))
             z_variable = float(match.group(3))
-            print(f"x: {x_variable}")
-            print(f"y: {y_variable}")
-            print(f"z: {z_variable}")
+            print(f"x box coordinate: {x_variable}")
+            print(f"y box coordinate: {y_variable}")
+            print(f"z box coordinate: {z_variable}")
         else:
             print("No x, y, or z variables found in the text.")
             x_variable = 0.0
             y_variable = 0.0
             z_variable = 0.0
+        return x_variable, y_variable, z_variable
 
-        sleep(30)
+    def test_box_moved(self, proc_output, demo):
+        """
+        Test case to see if box has moved
+        """
 
-        command = ["gz", "model", "-m", "artefacts_box", "-i"]
-        result = subprocess.run(command, check=True, text=True, stdout=subprocess.PIPE)
-        result_output = result.stdout
+        sleep(10)
 
-        # Extract new position data if it exists
-        position_pattern = r"pose\s*{\s*position\s*{\s*x:\s*(-?\d+\.\d+)\s*y:\s*(-?\d+\.\d+)\s*z:\s*(-?\d+\.\d+)"
+        (
+            TestCollision.x_variable,
```

Comment:
I see that you are using those values assigned to the class in the other test, but this somehow feels strange to me

Reply:
This was the only way I could figure out how to use values present in another class without re-initializing the class.